icubam/backoffice/translations/fr.csv

```diff
@@ -35,4 +36,63 @@
 "second","secondes","plural"
 "{delta} {units} ago","il y a {delta} {units}"
 "never","jamais"
-"now","à l'instant"
\ No newline at end of file
+"now","à l'instant"
+"Messages","Messages"
+"Access Tokens","Clés API"
+"Operational dashboard","Evolutions"
+"Map","Carte"
+"Regions","Régions"
+"ICUs","Réanimations"
+"Users","Utilisateurs",
+"Bed Counts","Stats"
+"Logout","Se déconnecter"
+"Home","Accueil"
+"Add","Ajouter"
+"name","Nom"
+"admin","Administrateur"
+"active","Actif"
+"created","Inséré"
+"icus","Réas"
+"manages","Gère"
+"city","Ville"
+"dept","Département"
+"region","Région"
+"regions","Régions"
+"users","Utilisateurs"
+"managers","Admins locaux"
+"access_key","Clé API"
+"is_active","Actif"
+"expiration_date","Expire"
+"access_type","type d'accès"
+"map","Carte"
+"db/bedcounts","Stats"
+"scheduled","Prochain envoi"
+"attempts /{}","Essais /{}"
+"first sent","Premier essai"
+"url","lien"
+"telephone","Téléphone"
```

Comment:
Maybe we should either have everything in lower or all Starting Upper?

Reply:
i would be in favor of a en.csv, to make sure things are always the way we want.